### PR TITLE
gradle-dependency-checker 1.0.0

### PR DESCRIPTION
**Pull request by imranMnts**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by imrekel** [comment]
hey, the `type_tags` field is not valid:
```
- (3/4) Running: Ensure all type_tags and project_type_tags are supported
 > Failed, error:
   - invalid type_tag: gradle
   - invalid type_tag: check
   - invalid type_tag: security
```
You can find the supported tags here: https://devcenter.bitrise.io/contributors/create-your-own-step/

File: steps/gradle-dependency-checker/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+title: gradle-dependency-check
+summary: |
+  OWASP dependency analysis for Gradle files
+description: |
+  Uses jeremylong/DependencyCheck to search Gradle for CVE vulnerabilites.
+  https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/index.html must be setup on the project before a first run
```
Comment (koral--) [inline steps/gradle-dependency-checker/1.0.0/step.yml:6]: It can use mardown links

File: steps/gradle-dependency-checker/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+source_code_url: https://github.com/odemolliens/bitrise-step-gradle-dependency-checker
+support_url: https://github.com/odemolliens/bitrise-step-gradle-dependency-checker/issues
+published_at: 2021-05-20T10:57:40.379921+02:00
+source:
+  git: https://github.com/odemolliens/bitrise-step-gradle-dependency-checker.git
+  commit: bea185995800ceab1280b1bcd71e5bed3c252623
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
```
Comment (koral--) [inline steps/gradle-dependency-checker/1.0.0/step.yml:20]: ios only?
It contradicts with gradle and react native.

File: steps/gradle-dependency-checker/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- dependency
+- gradle
+- check
+- security
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/gradle-dependency-checker/1.0.0/step.yml:31]: It does not seem to be actually used by script.

**Comment by imranMnts** [comment]
Thank you @koral-- @imrekel 
I applied your suggestions in this PR #3051

**Pull request closed without merging**